packages/suite/src/components/settings/SettingsSection.tsx

```diff
@@ -51,16 +60,20 @@ export const SettingsSection = ({
                 {!title && customHeader}
                 {title && !customHeader && (
                     <Title>
-                        {icon && <StyledIcon icon={icon} size={16} color={theme.TYPE_LIGHT_GREY} />}
-                        {title}
+                        {icon && <Icon icon={icon} size={24} color={theme.iconDefault} />}
+                        <H3>{title}</H3>
                     </Title>
                 )}
                 {description && !customHeader && (
-                    <Description type="label">{description}</Description>
+                    <Description>
+                        <Paragraph type="label">{description}</Paragraph>
```

Comment:
Simply style the `Paragraph`

Reply:
https://github.com/trezor/trezor-suite/pull/10050/commits/4e43da5f6ad6d8c577d90a35ae11cca64a284fe3